packages/editor/src/components/editor-canvas/index.js

```diff
@@ -304,7 +304,12 @@ function EditorCanvas(
 			height="100%"
 			iframeProps={ {
 				...iframeProps,
-				style: { ...iframeProps?.style, ...deviceStyles },
+				style: {
+					...iframeProps?.style,
+					marginLeft: 'auto',
+					marginRight: 'auto',
```

Comment:
Should we include that in the "deviceStyles" hook ?

Reply:
I don't have a strong opinion, but I think it's okay to include it in `deviceStyles` (`useResizeCanvas`). That is [this part](https://github.com/WordPress/gutenberg/blob/0456925145f6ca180639082ae83f4190b9e303ff/packages/block-editor/src/components/use-resize-canvas/index.js#L70). However, as I mentioned in the initial explanation, it doesn't exactly match the documentation, so I was just a little concerned.

> I first thought of adding default left and right auto margins to `useResizeCanvas` itself. However, the documentation [mentions](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/use-resize-canvas/README.md#useresizecanvas) that "it has no effect on the desktop".

Reply:
> However, the documentation [mentions](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/use-resize-canvas/README.md#useresizecanvas) that "it has no effect on the desktop".

I think we should remove that sentence from the documentation as this is more of an implementation detail.

Reply:
OK, Updated with [1eb867c](https://github.com/WordPress/gutenberg/pull/56970/commits/1eb867c99479e7398da8a118c6f88eef194d061f). I also removed that sentence from the document.